docs/user/introduction.asciidoc

```diff
@@ -43,24 +32,15 @@ you want to view existing dashboards and drill down into details.
 
 *{kib} works with all types of data.* Your data can be structured or unstructured text,
 numerical data, time-series data, geospatial data, logs, metrics, security events,
```

Comment:
This is one of the terms in https://brand.elastic.co/302f66895/p/194a3b-writing-style-guide
```suggestion
numerical data, time series data, geospatial data, logs, metrics, security events,
```